### DISCUSSION TITLE
What algorithm do you use to convert distance to percentage when verifying two images?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#873] [Q&A]
What algorithm do you use to convert distance to percentage when verifying two images?

**Comment by pospielov** [comment]
https://github.com/exadel-inc/CompreFace/blob/master/java/api/src/main/java/com/exadel/frs/core/trainservice/component/classifiers/EuclideanDistanceClassifier.java#L142
here is the formula. The coefficients are calculated from an experiment with 200 000 faces.

**Reply by GustavoDenobi** [reply]
Are these coefficients somehow accessible? How do I get them? Couldn't figure it out.

In my application, I'd like to be able to check if unknown faces match between them, so I need to directly calculate the similarity between the embeddings.

**Reply by pospielov** [reply]
If you use CompreFace, you can just use CompreFace`s similarity to match the faces.
If you created your own library, you could define this transformation by yourself in a different way, which is more convenient for you. The simplest is to apply MinMaxScaler from sklearn
